adapter/src/main/java/com/twilio/kudu/sql/CalciteKuduTable.java

```diff
@@ -234,18 +234,18 @@ public RelNode toRel(final RelOptTable.ToRelContext context, final RelOptTable r
    *                                a prefix of the primary key of the table, or
    *                                an empty list if the group by and order by
    *                                columns are the same
-   * @param sortPkColumnNames       the names of the primary key columns that are
-   *                                present in the ORDER BY clause
+   * @param sortPkColumns           the indexes of the primary key columns that
```

Comment:
any reason for changing from indexName to index values?

Reply:
Thats how it will be used in the end to sort he rows returned 
https://github.com/twilio/calcite-kudu/blob/71991a984e0c7357a2d0909741da675b264a65b7/adapter/src/main/java/com/twilio/kudu/sql/CalciteRow.java#L30
Instead of storing these as a list of column name and then looking up the indexes in the projection schema, this pr just passes them as a list of column indexes.